src/status_im/contexts/onboarding/syncing/progress/view.cljs

```diff
@@ -25,17 +26,32 @@
     :title-accessibility-label       :progress-screen-title
     :description-accessibility-label :progress-screen-sub-title}])
 
+(defn- navigate-to-enter-seed-phrase
+  []
+  (rf/dispatch [:navigate-back-to :screen/onboarding.sync-or-recover-profile])
+  (debounce/debounce-and-dispatch
```

Comment:
why is only this event debounced?

Reply:
Thank you, @flexsurfer, for reviewing the PR and spotting this issue. The intention was to use setTimeout to add a delay between navigation, but debounce was used by mistake. (fixed now)

Reply:
Thank you, @flexsurfer, for reviewing the PR. We are using debounce-and-dispatch here to introduce a delay between navigating back to the `sync-or-recover-profile` screen and moving to the `sign-in-by-seed-phrase` screen.